src/test/java/net/sf/jabref/importer/fileformat/MedlinePlainImporterTest.java

```diff
@@ -0,0 +1,122 @@
+package net.sf.jabref.importer.fileformat;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.bibtex.BibtexEntryAssert;
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.model.entry.BibEntry;
+
+/**
+ * Test class for the MedlinePlain importer.
+ */
+
+public class MedlinePlainImporterTest {
+
+    private MedlinePlainImporter importer;
+
+
+    @Before
+    public void setUp() throws Exception {
+        Globals.prefs = JabRefPreferences.getInstance();
+        importer = new MedlinePlainImporter();
+    }
+
+    @Test
+    public void testIsRecognizedFormat() throws Exception {
+
+        List<String> list = Arrays.asList("CopacImporterTest1.txt", "CopacImporterTest2.txt", "IsiImporterTest1.isi",
+                "IsiImporterTestInspec.isi", "IsiImporterTestWOS.isi", "IsiImporterTestMedline.isi",
+                "MedlinePlainImporterTestEmpty.txt");
+
+        //Test with txt Files which are not in recognized Format
+        for (String str : list) {
+            try (InputStream is = MedlinePlainImporter.class.getResourceAsStream(str)) {
+                Assert.assertFalse(importer.isRecognizedFormat(is));
+            }
+        }
+
+        list = Arrays.asList("MedlinePlainImporterTest.txt", "MedlinePlainImporterTest1.txt",

```

Comment:
Yes, you are right, normally one should be enough. But have a look at the `isRecognized` method and see if you cover all cases. Also it does not harm to check edge cases, like empty files.